### ISSUE TITLE
Create Wall Loss Chamber Simulation Example in docs/Examples

### ISSUE THREAD

**Issue by Gorkowski** [uncscode/particula#820]
**Parent Issue:** #72

**Dependencies:**
- #816 - WallLossStrategy ABC and SphericalWallLossStrategy
- #817 - RectangularWallLossStrategy
- #818 - Wall Loss Builders and Factory
- #819 - WallLoss Runnable Process

---

## Problem/Motivation

With the wall loss system fully implemented (#816-#819), we need to create comprehensive documentation and examples showing how to use wall loss in chamber simulations. This will help users understand how to integrate wall loss into their particle simulations.

**Parent Issue:** #72 - Implement Wall Loss Strategies and Builders for All Particle Representations

**Dependencies:**
- #816 - WallLossStrategy ABC and SphericalWallLossStrategy
- #817 - RectangularWallLossStrategy  
- #818 - Wall Loss Builders and Factory
- #819 - WallLoss Runnable Process (ALL MUST BE COMPLETED FIRST)

**Value:**
- Provides practical examples for users
- Demonstrates integration with existing dynamics (coagulation, condensation)
- Shows comparison between chamber geometries
- Serves as documentation and tutorial for wall loss feature

### Example Notebook
- [ ] Create `docs/Examples/Chamber_Wall_Loss/Wall_Loss_Tutorial.ipynb`
- [ ] Include introduction to wall loss physics
- [ ] Demonstrate spherical chamber wall loss
- [ ] Demonstrate rectangular chamber wall loss
- [ ] Show comparison between geometries
- [ ] Show process chaining with coagulation
- [ ] Include visualization of concentration decay

### Content Sections
- [ ] Introduction: Wall loss physics and importance
- [ ] Setup: Creating particles and atmosphere
- [ ] Basic Usage: Simple wall loss simulation
- [ ] Geometry Comparison: Spherical vs rectangular
- [ ] Advanced: Combining with coagulation and condensation
- [ ] Analysis: Plotting and interpreting results

### Documentation Updates
- [ ] Update `docs/Examples/index.md` to include new example
- [ ] Create `docs/Examples/Chamber_Wall_Loss/index.md` if needed

## Technical Context

**New Files to Create:**
- `docs/Examples/Chamber_Wall_Loss/Wall_Loss_Tutorial.ipynb` (~300-400 lines)
- `docs/Examples/Chamber_Wall_Loss/index.md` (~20 lines)

**Files to Modify:**
- `docs/Examples/index.md` - Add link to new example

**Related Examples:**
- `docs/Examples/Dynamics/` - Pattern for dynamics examples
- `docs/Examples/Aerosol/Aerosol_Tutorial.ipynb` - Notebook format reference

### Notebook Structure

```markdown
# Wall Loss in Chamber Simulations

## Introduction

Wall loss is an important process in environmental chamber experiments...

## Setup

### Import Libraries
```python
import numpy as np
import matplotlib.pyplot as plt
import particula as par
```

### Create Particle Distribution
```python
# Create a lognormal particle distribution
particle = (
    par.particles.PresetParticleRadiusBuilder()
    .set_mode(np.array([100e-9]))  # 100 nm mode diameter
    .set_geometric_standard_deviation(np.array([1.5]))
    .set_number_concentration(np.array([1e10]))  # 10^10 #/m³
    .set_density(1500.0)  # kg/m³
    .build()
)
```

### Create Atmosphere
```python
atmosphere = (
    par.gas.AtmosphereBuilder()
    .set_temperature(298.15)  # K
    .set_pressure(101325.0)  # Pa
    .build()
)
```

### Build Aerosol
```python
aerosol = (
    par.AerosolBuilder()
    .add_particle(particle)
    .set_atmosphere(atmosphere)
    .build()
)
```

### Create Strategy and Process
```python
# Create spherical wall loss strategy
spherical_strategy = par.dynamics.SphericalWallLossStrategy(
    wall_eddy_diffusivity=0.001,  # s⁻¹, typical value
    chamber_radius=0.5,  # 0.5 m radius
    distribution_type="discrete",
)

# Create runnable process
wall_loss = par.dynamics.WallLoss(wall_loss_strategy=spherical_strategy)
```

### Simulate Wall Loss Over Time
```python
# Simulation parameters
total_time = 3600  # 1 hour in seconds
time_step = 60  # 1 minute steps
times = np.arange(0, total_time + time_step, time_step)

# Track concentration over time
concentrations = [aerosol.particles.get_concentration().sum()]

# Run simulation
for t in times[1:]:
    aerosol = wall_loss.execute(aerosol, time_step=time_step)
    concentrations.append(aerosol.particles.get_concentration().sum())

# Plot results
plt.figure(figsize=(10, 6))
plt.plot(times / 60, concentrations, 'b-', linewidth=2)
plt.xlabel('Time (minutes)')
plt.ylabel('Total Concentration (#/m³)')
plt.title('Particle Wall Loss in Spherical Chamber')
plt.grid(True)
plt.yscale('log')
plt.show()
```

### Setup Both Strategies
```python
# Reset aerosol for fair comparison
aerosol_spherical = build_aerosol()  # Helper function
aerosol_rectangular = build_aerosol()

# Spherical chamber (0.5 m radius, volume ≈ 0.52 m³)
spherical_strategy = par.dynamics.SphericalWallLossStrategy(
    wall_eddy_diffusivity=0.001,
    chamber_radius=0.5,
)

# Rectangular chamber with same volume (1m x 0.72m x 0.72m ≈ 0.52 m³)
rectangular_strategy = par.dynamics.RectangularWallLossStrategy(
    wall_eddy_diffusivity=0.001,
    chamber_dimensions=(1.0, 0.72, 0.72),
)

wall_loss_spherical = par.dynamics.WallLoss(spherical_strategy)
wall_loss_rectangular = par.dynamics.WallLoss(rectangular_strategy)
```

### Run Comparison
```python
conc_spherical = []
conc_rectangular = []

for t in times:
    conc_spherical.append(aerosol_spherical.particles.get_concentration().sum())
    conc_rectangular.append(aerosol_rectangular.particles.get_concentration().sum())
    
    if t < total_time:
        aerosol_spherical = wall_loss_spherical.execute(
            aerosol_spherical, time_step=time_step
        )
        aerosol_rectangular = wall_loss_rectangular.execute(
            aerosol_rectangular, time_step=time_step
        )

# Plot comparison
plt.figure(figsize=(10, 6))
plt.plot(times / 60, conc_spherical, 'b-', label='Spherical', linewidth=2)
plt.plot(times / 60, conc_rectangular, 'r--', label='Rectangular', linewidth=2)
plt.xlabel('Time (minutes)')
plt.ylabel('Total Concentration (#/m³)')
plt.title('Wall Loss Comparison: Spherical vs Rectangular Chamber')
plt.legend()
plt.grid(True)
plt.yscale('log')
plt.show()
```

### Wall Loss + Coagulation
```python
# Create coagulation strategy
coag_strategy = par.dynamics.BrownianCoagulationStrategy(
    distribution_type="discrete"
)
coagulation = par.dynamics.Coagulation(coagulation_strategy=coag_strategy)

# Chain processes: coagulation then wall loss
combined_process = coagulation | wall_loss

# Compare with wall loss only
aerosol_wall_only = build_aerosol()
aerosol_combined = build_aerosol()

conc_wall_only = []
conc_combined = []

for t in times:
    conc_wall_only.append(aerosol_wall_only.particles.get_concentration().sum())
    conc_combined.append(aerosol_combined.particles.get_concentration().sum())
    
    if t < total_time:
        aerosol_wall_only = wall_loss.execute(
            aerosol_wall_only, time_step=time_step
        )
        aerosol_combined = combined_process.execute(
            aerosol_combined, time_step=time_step
        )

# Plot
plt.figure(figsize=(10, 6))
plt.plot(times / 60, conc_wall_only, 'b-', label='Wall Loss Only')
plt.plot(times / 60, conc_combined, 'g--', label='Coagulation + Wall Loss')
plt.xlabel('Time (minutes)')
plt.ylabel('Total Concentration (#/m³)')
plt.title('Effect of Combined Processes')
plt.legend()
plt.grid(True)
plt.yscale('log')
plt.show()
```

## Using the Builder Pattern

```python
# Using builders for cleaner configuration
strategy = (
    par.dynamics.SphericalWallLossBuilder()
    .set_wall_eddy_diffusivity(0.001, "1/s")
    .set_chamber_radius(50, "cm")  # Unit conversion
    .set_distribution_type("discrete")
    .build()
)

# Using factory for strategy selection
factory = par.dynamics.WallLossFactory()
strategy = factory.get_strategy(
    strategy_type="spherical",
    parameters={
        "wall_eddy_diffusivity": 0.001,
        "chamber_radius": 0.5,
    }
)
```

## Size-Dependent Wall Loss Analysis

```python
# Analyze wall loss rate vs particle size
sizes = np.logspace(-9, -6, 50)  # 1 nm to 1 µm

# Calculate loss coefficient for each size
loss_coefficients = []
for size in sizes:
    # Create single-size particle
    particle = create_single_size_particle(size)
    coeff = spherical_strategy.loss_coefficient(
        particle, temperature=298.15, pressure=101325.0
    )
    loss_coefficients.append(coeff)

# Plot
plt.figure(figsize=(10, 6))
plt.loglog(sizes * 1e9, loss_coefficients, 'b-', linewidth=2)
plt.xlabel('Particle Diameter (nm)')
plt.ylabel('Loss Coefficient (s⁻¹)')
plt.title('Size-Dependent Wall Loss Coefficient')
plt.grid(True, which="both", ls="-", alpha=0.5)
plt.show()
```

## Summary

Key takeaways:
1. Wall loss is size-dependent (diffusion dominates for small particles, settling for large)
2. Chamber geometry affects loss rates (surface area to volume ratio)
3. Wall loss can be combined with other processes using process chaining
4. Use builders for clean configuration and unit conversion
```

### Update index.md

Add to `docs/Examples/index.md`:

```markdown
## Chamber Wall Loss

- [Wall Loss Tutorial](Chamber_Wall_Loss/Wall_Loss_Tutorial.ipynb) - 
  Simulating particle wall deposition in environmental chambers
```

## Suggested Tests

The notebook should be testable by running all cells without errors:

- [ ] All imports work correctly
- [ ] Particle and atmosphere creation succeeds
- [ ] Wall loss calculations produce reasonable values
- [ ] Plots render correctly
- [ ] Combined processes execute without errors

## Success Criteria

- [ ] Notebook runs without errors from start to finish
- [ ] All code examples are correct and follow best practices
- [ ] Visualizations clearly demonstrate concepts
- [ ] Documentation explains physics and usage clearly
- [ ] Index files updated with links to new content
- [ ] Example follows existing notebook format conventions

## Edge Cases and Considerations

- **Long simulation times**: Use appropriate time steps
- **Very small particles**: May have numerical issues
- **Unit consistency**: Always specify units clearly
- **Process order**: Document recommended order for chaining

## References

**Parent Issue:**
- #72 - Implement Wall Loss Strategies and Builders

**Dependencies:**
- #816 - WallLossStrategy ABC and SphericalWallLossStrategy
- #817 - RectangularWallLossStrategy
- #818 - Wall Loss Builders and Factory
- #819 - WallLoss Runnable Process

**Related Examples:**
- `docs/Examples/Dynamics/` - Existing dynamics examples
- `docs/Examples/Aerosol/Aerosol_Tutorial.ipynb` - Notebook format reference

**Scientific References:**
- Crump & Seinfeld (1981), J. Aerosol Sci.
- McMurry & Rader (1985), Aerosol Sci. Tech.
